Dockerfile

```diff
@@ -47,6 +48,11 @@ RUN pecl install igbinary-1.2.1 && \
     echo 'extension=igbinary.so' > /etc/php5/mods-available/igbinary.ini && \
     php5enmod igbinary
 
+RUN printf "\n" | pecl install apcu-4.0.7 && \
```

Comment:
required for next-gen phystrix work, and well as an on-instance cache for most advanced work